README.rst

```diff
@@ -262,6 +263,24 @@ foo
 bar
 
 
+Computing and Retrieving Checksums
+----------------------------------
+
+Each data object may be associated with a checksum by calling chksum() on the object in question.  Various 
+behaviors can be elicited by passing in combinations of keywords (for a description of which, please consult the
+`header documentation <https://docs.irods.org/4.2.10/doxygen/dataObjChksum_8h.html>`_ .)
```

Comment:
let's see if we can get a non-version-specific link... hmmm... just point to the source in 4-2-stable?  not sure...

Reply:
Done

Reply:
dubious

Reply:
Right, looking for a better link .

Reply:
dubious because of doxygen formatting?

Reply:
Not sure if that's a reference to the Doxygen formatting or that I missed a change ....

Reply:
i was dubious of your 'done'.   now it's changed.   resolved!

Reply:
ah, good ...